packages/core/src/components/navigation/Navbar/Navbar.tsx

```diff
@@ -26,19 +34,18 @@ function Navbar() {
   }
 
   return (
-    <header
-      data-fs-navbar
+    <UINavbar
+      className="layout__content-full"
       data-fs-navbar-scroll={scrollDirection}
```

Comment:
Can we transform this data-attribute in prop on @faststore/components?

Reply:
AFAIK we gonna keep the use of this `layout` global classes for now, right @renatamottam @hellofanny ?